### ISSUE TITLE
Export list not maintaining filename or template

### ISSUE THREAD

**Issue by dougK44883** [jessedp/tablo-tools-electron#132]
**Describe the bug**
Select show(s) or episodes for export. Use a different default template.  Select show(s) or episode, adding the to the cart/export list.  Repeat a few times maybe.  Select Export from cart.  Every selected show/entry in the cart/export list gets the newly chosen default template for the filename - all are using the same template.

Previous version 0.2.4 would maintain template/filename once entry was added to the cart/export list.  Changing default template would **not** change previously selected entries.  Benefit of multiple naming templates was not limited to per-session.

**To Reproduce**
(see "Describe the bug)

**Expected behavior**

-Maintain filename between changing templates. 

**Screenshots**
Screenshot from v0.2.4 - several shows added to export list, changing default template between each selection.  This methodology no longer functions via v0.3.4  -  each template change will change filename of previously selected.
![image](https://user-images.githubusercontent.com/42115699/195987319-1273a9ee-6635-47a4-9e3b-f65c8703d664.png)

----
Screenshot from v.0.3.4 - same shows added, using same template when show was selected/added to cart/export list.  Each changed to the last selected template.  
![image](https://user-images.githubusercontent.com/42115699/195987659-b2d391b8-4888-466c-9665-e29b4c1564fe.png)

Note, they vary due to the network name in the template, not because they are actually different beyond this.  Despite some similarities, the 0.2.4 are distinct and if necessary can be posted for clarity.
**Environment:**
 - OS:  Debian 11 Bullseye
 - App Version from the Home screen v0.3.4 
 
**Additional context**

Having multiple template is an awesome feature of Tablo Tools.  Not being able to use more than one per-session tends to negate their benefits.

**Comment by dougK44883** [comment]
This has never gotten competently fixed.  It appeared random or occasional and I just dealt with it.  
Summery:  Once you add recordings to the export cart - change the default template, sometimes some of the already selected recordings will change from their previously selected naming template to the newly selected on.

I haven't fully investigated, I may have found a pattern.  It may be the last or previously selected series -select all- added to the cart will get changed to the newly chosen template.  Even funnier, it only happens the first time changing the template.

I know it happens, usually once.  Likely the first change.

**Comment by dougK44883** [comment]
May be limited to setting default from file editor.  Change default template from file editor, add recordings.  Check cart, the last previously added [set of] recordings retroactively change to the newly selected default template.
Appears to be a one-time deal.  Subsequent changes to default template works as expected, just the first change via file editor flubs things up.

**Comment by jessedp** [comment]
Do you visit the Export List in between changing things? Currently nothing is locked in place until you view it in the Export List. I may have to rework that again so it happens the moment the recording hits the cart.

**Comment by dougK44883** [comment]
I recall an issue since the overhaul with v0.3.Xx  We reached a point it was working with minor annoyances.  Figured if no one else had issues I moved on.  Recently I believed I may have noticed a pattern, hadn't given it a full in depth investigation. 

Yes, since I know at some point, occasionally show filenames would periodically get changed I would check the export list from time to time.  I didn't want to get in too far before realizing things weren't going as expected.

I've noticed it seems to only happen one, maybe.  Generally, maybe, only after changed default template the first time.  It might only be an issue changing default from the file editor instead of the "settings" page.
